src/server.h

```diff
@@ -2595,10 +2594,20 @@ int serverCommunicateSystemd(const char *sd_notify_msg);
 void serverSetCpuAffinity(const char *cpulist);
 void dictVanillaFree(dict *d, void *val);
 
+/* ERROR STATS constants */
+#define ERROR_STATS_LUA_LIMIT                                                                                          \
+    128 /* After the errors RAX reaches this limit, instead of tracking                                                \
+          custom LUA errors, we track the error under the symbol below. */
```

Comment:
```suggestion
/* After the errors RAX reaches this limit, instead of tracking
 * custom LUA errors, we track the error under the symbol below. */
#define ERROR_STATS_LUA_LIMIT 128
```
This is kind of silly formatting.

Reply:
This was suggested by the clang check formatter. So I made the change here: https://github.com/valkey-io/valkey/pull/500/commits/05e946fc92dc5d034e5982bcb88f752d214e0f49

Reply:
Clang-format can accept multiple ways of writing, but I don't think we should accept everything everything that clang-format suggests to us. :)

Reply:
I reverted this format change and the clang check failed on these errors (on the highlighted part above). Let me know if this is ok, or if this needs to be addressed